specs/tables.md

```diff
@@ -45,9 +45,14 @@ Type sizes:
 > - **memoryAddress**: 40 bytes
 > - **Memory -> value, valuePrev**: 1 byte
 > - **storageKey**: field size, RLC encoded (Random Linear Combination)
-> - **value, valuePrev**: variable size, depending on Key 0 (Tag) and Key 3 where appropiate.
-> - **Key2** is reserved for Ethereum Address key type
-> - **Key4** is reserved for RLC encoded key type
+
+- **value, valuePrev**: variable size, depending on Key 0 (Tag) and Key 3 where appropiate.
+- **Key2** is reserved for Ethereum Address key type
+- **Key4** is reserved for RLC encoded key type
+- **logID**, 8 bytes
```

Comment:
Could you please specify if logID is unique per block?  And maybe add a note that this is not yet stable (as I think in the end logID will be unique per Tx, so that we know the log position within the receipt).

Reply:
yes, logID will be unique per Tx.